std_msgs/Header header
string topic_name
string topic_arg
float32 resolution
float32 n_rows
float32 n_cols
float32[] data
